recipe/build.sh

```diff
@@ -1,4 +1,10 @@
 #! /bin/bash
 
-cmake -DCMAKE_INSTALL_PREFIX=$PREFIX -DBUILD_SHARED_LIBS=y . -Bbuilddir
+cmake . \
+-DCMAKE_INSTALL_PREFIX=$PREFIX \
+-DBUILD_SHARED_LIBS=y \
```

Comment:
maybe instead of `y` also use ON or TRUE?

Reply:
I just copy/pasted from linux existing build script but if you prefer I can also change that

Reply:
yep would be nice!